packages/react-conformance/src/types.ts

```diff
@@ -1,9 +1,16 @@
-export interface IsConformantOptions {
-  constructorName: string;
-  tests: String[];
-  requiredProps?: object;
-  exportedAtTopLevel: boolean;
-  // packageRoot: any;
+import * as React from 'react';
+import { ComponentDoc } from 'react-docgen-typescript';
+
+export interface TestingOptions<TProps = {}> {
+  componentPath: string;
+  Component: React.ComponentType<TProps>;
```

Comment:
You should add `displayName` here too (I think that's better than `constructorName` since it's more generic) so you can reference the expected value in the tests.